homeassistant/components/blink/camera.py

```diff
@@ -76,7 +76,7 @@ async def async_enable_motion_detection(self) -> None:
             await self._coordinator.async_refresh()
 
         except asyncio.TimeoutError:
-            self._attr_available = False
+            self.coordinator.last_update_success = False
```

Comment:
Hold up, this is not a change you want to make. This field is representing if the coordinator did a good job of executing his part. You're catching a Timeout error here, is that for the `async_arm` or the `async_refresh`? The coordinator handles timeout itself, so if that fails it will set this variable itself.

Reply:
OK - I suppose we should just raise an error here telling HA the command failed... simple as that.

Reply:
To add to this, can we make the try block smaller so only the lines that could raise the error are in there and the rest is out?